worlds/sc2/item_descriptions.py

```diff
@@ -716,6 +716,7 @@ def _ability_desc(unit_name_plural: str, ability_name: str, ability_description:
     item_names.MUTALISK_CORRUPTOR_DEVOURER_ASPECT: "Anti-air flyer. Attack inflict Acid Spores. Can attack air units. Morphed from the Mutalisk or Corruptor.",
     item_names.ROACH_RAVAGER_ASPECT: "Ranged artillery. Can use Corrosive Bile. Can attack ground units. Morphed from the Roach.",
     item_names.ROACH_PRIMAL_IGNITER_ASPECT: "Assault unit. Has an area-damage attack. Regenerates life quickly when burrowed. Can attack ground units. Morphed by merging two Roaches.",
+    item_names.NYDUS_WORM: "Friendly ground units can use a Nydus Worm to instantly travel to any other Nydus Worm or Nydus Network owned by the player. Grows creep used to feed nearby Zerg structures.",

```

Comment:
I recommend a 2~4 word sentence to introduce the unit, to match the pattern set by other descriptions. Something like
> Long-range transport network. Nydus Worms and Nydus Networks can load friendly ground units to be unloaded to any other Nydus structure on the map.

Also consider if Nydus worm creep spread should be a separate upgrade. I'd be happy with it as baseline, myself. You wrote elsewhere that you wanted to make Omega Worms an upgrade, and I'd recommend splitting it up into several ugprades (free nydus worms, instant-build nydus worms, detector is weak on its own and can be paired with something else like fast-unload or health buff or creep spread)